airflow/utils/log/file_processor_handler.py

```diff
@@ -144,6 +144,6 @@ def _init_file(self, filename):
                     raise
 
         if not os.path.exists(full_path):
-            open(full_path, "a").close()
+            open(os.path.abspath(full_path), "a").close()
```

Comment:
Do we need this though??

https://github.com/apache/airflow/blob/05a195c11b1a75c193ae172263642fe80f40025f/airflow/utils/log/file_processor_handler.py#L136

https://github.com/apache/airflow/blob/05a195c11b1a75c193ae172263642fe80f40025f/airflow/utils/log/file_processor_handler.py#L95-L98

https://github.com/apache/airflow/blob/master/airflow/config_templates/airflow_local_settings.py#L78

Reply:
@kaxil Not sure I follow. The clients of this method expect to receive the relative path.

Reply:
`child_process_log_directory` config setting, should, normally be an absolute path.

Perhaps there should be a validation/transform step (at config load time) that sets it to an absolute path if it isn't?

Reply:
Yeah, cos a variable called "full_path" sure sounds like it was expected to be an absolute path already.